Arduino/victoria_teensy_ros_node/RosParamHelper.h

```diff
@@ -0,0 +1,80 @@
+#ifndef ROS_PARAM_HELPER_h
+#define ROS_PARAM_HELPER_h
+
+#include <ros.h>
+
+class RosParamHelper {
+
+public:
+  RosParamHelper(ros::NodeHandle nh);
+  
+  /**
+   * Retrieves int parameter from ros and sets it
+   * into the param.
+   * 
```

Comment:
`\param` should be listed when specifying input params.
`\param[out]` for output parameters.